packages/devextreme-cli/testing/linting.js

```diff
@@ -0,0 +1,61 @@
+const reactEnv = require('./env.react');
+const vueEnv = require('./env.vue');
+const angularEnv = require('./env.angular');
+const minimist = require('minimist');
+const fs = require('fs');
+const ESLint = require('eslint').ESLint;
+
+const args = minimist(process.argv.slice(), {
```

Comment:
It looks like a duplication of code. Maybe it's better to extract some utility that will get an array of engines and filter it by the argument? Something like this:
```js
const environments = [
  require('./env.react'),
  require('./env.vue'),
  require('./env.angular')
];
filterEnvs(environments, provess.argv).forEach((env) => {
  ...
})
```